checkoutmanager/runner.py

```diff
@@ -82,6 +82,21 @@ def execute_action(dirinfo, custom_actions, action):
         return e
 
 
+def run(action, group=None, custom_actions=None, conf=None, single=False):
+
+    if not conf:
```

Comment:
``conf `` needs to be passed in here, since main() may get something else from the user. ``custom_actions`` doesn't seem necessary, though. I'll trim that down.